frontend/packages/core/src/AppLayout/user.tsx

```diff
@@ -33,6 +33,10 @@ const Initials = styled(Typography)`
   `}
 `;
 
+interface JwtToken {
```

Comment:
My typescript skills are pretty weak but do they have an interface which we can just use? or do we have to define it ourselves?

Reply:
Not right now. [The existing type definitions](https://github.com/auth0/jwt-decode/blob/master/index.d.ts) return unknown but I can see us adding at least the [registered claim names](https://tools.ietf.org/html/rfc7519#section-4.1) as optional.